pkg/teleporter/relayer.go

```diff
@@ -124,19 +125,24 @@ func RelayerIsUp(runFilePath string) (bool, int, *os.Process, error) {
 	if err := json.Unmarshal(bs, &rf); err != nil {
 		return false, 0, nil, err
 	}
-	proc, err := os.FindProcess(rf.Pid)
+	proc, err := GetProcess(rf.Pid)
 	if err != nil {
 		// after a reboot without network cleanup, it is expected that the file pid will exist but the process not
-		err := removeRelayerRunFile(runFilePath)
-		return false, 0, nil, err
+		return false, 0, nil, removeRelayerRunFile(runFilePath)
+	}
+	return true, rf.Pid, proc, nil
+}
+
+func GetProcess(pid int) (*os.Process, error) {
```

Comment:
should be in utils

Reply:
this is going to live in a package that is to be imported from SDK. for the moment keeping it here